src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -380,29 +321,22 @@ class ContentSidebar extends React.PureComponent<Props, State> {
             onVersionHistoryClick,
         }: Props = this.props;
         const { file, isLoading, isOpen, metadataEditors }: State = this.state;
-        const hasSidebar = SidebarUtils.shouldRenderSidebar(this.props, file, metadataEditors);
 
-        if (!file || !hasSidebar || !fileId) {
+        if (!file || !fileId || !SidebarUtils.shouldRenderSidebar(this.props, file, metadataEditors)) {
             return null;
         }
 
-        const selectedView = this.getSidebarView();
         const hasSkills = SidebarUtils.shouldRenderSkillsSidebar(this.props, file);
         const hasDetails = SidebarUtils.canHaveDetailsSidebar(this.props);
         const hasMetadata = SidebarUtils.shouldRenderMetadataSidebar(this.props, metadataEditors);
-        const styleClassName = classNames(
-            'be bcs',
-            {
-                [selectedView ? `bcs-${selectedView}` : '']: isOpen,
-                'bcs-is-open': isOpen,
-            },
-            className,
-        );
+        const styleClassName = classNames('be bcs', className, {
+            'bcs-is-open': isOpen,
+        });
 
         return (
             <Internationalize language={language} messages={messages}>
-                <aside id={this.id} className={styleClassName}>
-                    <div className="be-app-element">
+                <MemoryRouter initialEntries={[`/${defaultView || ''}`]}>
```

Comment:
If we don't have a default view passed in, will the memory router remember the previously active tab? Or does navigating in preview destroy the router?

Reply:
Navigating a file collection doesn't update the "route" so the previous sidebar remains. Closing and reopening Preview destroys the router, but the behavior is the same as today.